service/controller/controller.go

```diff
@@ -1,133 +0,0 @@
-package controller
```

Comment:
Changing the controller magic to align with the other operators. This shit here was some aweful outdate.